sidecred.go

```diff
@@ -20,6 +20,14 @@ type CredentialRequest struct {
 	// name is up to the individual provider.
 	Name string `json:"name"`
 
+	// Rotation is an override for the default rotation window
+	// measured in minutes.
```

Comment:
I think the durations that are exposed in the configurations are all in seconds. It seems like this should be the same, for consistency? The end user isn't really exposed to the default credential rotation window setting as `10m` anyway.

Reply:
Great catch 👍 Going forward I think it makes sense to switch to using `time.Duration` and supporting e.g. `15m` instead of 900 since its much more user friendly. Created an issue here now: https://github.com/telia-oss/sidecred/issues/59 - thoughts?

Reply:
I agree that it makes for a better user experience and falls in line with other system like Kubernetes (for health checks)